### DISCUSSION TITLE
Geometry not found error

### DISCUSSION THREAD

**Discussion by MonikaShah** [opengisch/QField#3908] [Q&A]
Building points are all marked in the project. 
All goes well in the starting when an update is made on building name etc details.
After few days of continual usage of the project, 
1. changes are not applied , saying geometry not found. How can this happen? 
2. Changes are applied, after syncronisation , changes are lost. 
and also saying geometry not found.
When the same thing is updated using QGIS and updated to cloud, it works all well.
![Screenshot from 2023-02-15 12-17-55](https://user-images.githubusercontent.com/4188149/218953405-1dcb3f53-547b-46d5-a59f-3f306dcc4aa0.png)
![Screenshot from 2023-02-15 12-17-30](https://user-images.githubusercontent.com/4188149/218953409-1fa514c4-c4c3-44d0-9f82-f96370ed8975.png)
![Screenshot from 2023-02-15 12-16-54](https://user-images.githubusercontent.com/4188149/218953423-4a987a73-1a5f-4e54-820a-302f25a63a03.png)
![Screenshot from 2023-02-15 12-16-45](https://user-images.githubusercontent.com/4188149/218953431-4528a3ed-023e-44cf-801b-69c8477cd713.png)

Proejcts :

**Comment by MonikaShah** [comment]
Sorry , it says Geometry not modified in this case.

**Comment by MonikaShah** [comment]
Why changes are not applied in cases like this.  Project : communitygis/Nward_5jan23
![Screenshot from 2023-02-15 12-22-24](https://user-images.githubusercontent.com/4188149/218954024-c2a5c136-24b0-4e89-ac74-3b5cc5b96690.png)

**Comment by stempest123** [comment]
Did you manage to fix this issue? I am experiencing this today
![image](https://user-images.githubusercontent.com/128692753/227174223-f52c7ff9-8ed4-4bb6-b207-f52fb5ea51c1.png)

**Comment by carlosonecastellanos** [comment]
Did you manage to fix this issue?
